aws-network-acl-allows-public-ingress.tf

```
@@ -0,0 +1,1439 @@
+
+  # ruleid: aws-network-acl-allows-public-ingress
+  ingress {
+    rule_no    = 1
+    protocol   = "tcp"
+    from_port  = 22
+    to_port    = 22
+    action     = "allow"
+    cidr_block = "192.168.255.255/15" # 192.168.0.0 - 192.169.255.255
+  }
+}
+
+# ruleid: aws-network-acl-allows-public-ingress
+resource "aws_network_acl_rule" "fail_end_outside_private_network_12" {
+  egress      = false
+  protocol    = "tcp"
+  from_port   = 22
+  to_port     = 22
+  rule_action = "allow"
+  cidr_block  = "192.169.0.0/15" # 192.168.0.0 - 192.169.255.255
+}
+
+resource "aws_network_acl" "fail_end_outside_private_network_12" {
+  vpc_id = aws_vpc.example.id
+
+  # ruleid: aws-network-acl-allows-public-ingress
+  ingress {
+    rule_no    = 1
+    protocol   = "tcp"
+    from_port  = 22
+    to_port    = 22
+    action     = "allow"
+    cidr_block = "192.169.0.0/15" # 192.168.0.0 - 192.169.255.255
+  }
+}
+
+# ruleid: aws-network-acl-allows-public-ingress
+resource "aws_network_acl_rule" "fail_end_outside_link_local_1" {
+  egress      = false
+  protocol    = "tcp"
+  from_port   = 22
+  to_port     = 22
+  rule_action = "allow"
+  cidr_block  = "169.254.0.0/15" # 169.254.0.0 - 169.255.255.255
+}
+
+resource "aws_network_acl" "fail_end_outside_link_local_1" {
+  vpc_id = aws_vpc.example.id
+
+  # ruleid: aws-network-acl-allows-public-ingress
+  ingress {
+    rule_no    = 1
+    protocol   = "tcp"
+    from_port  = 22
+    to_port    = 22
+    action     = "allow"
+    cidr_block = "169.254.0.0/15" # 169.254.0.0 - 169.255.255.255
+  }
+}
+
+# ruleid: aws-network-acl-allows-public-ingress
+resource "aws_network_acl_rule" "fail_end_outside_link_local_2" {
+  egress      = false
+  protocol    = "tcp"
+  from_port   = 22
+  to_port     = 22
+  rule_action = "allow"
+  cidr_block  = "169.254.99.99/15" # 169.254.0.0 - 169.255.255.255
+}
+
+resource "aws_network_acl" "fail_end_outside_link_local_2" {
+  vpc_id = aws_vpc.example.id
+
+  # ruleid: aws-network-acl-allows-public-ingress
+  ingress {
+    rule_no    = 1
+    protocol   = "tcp"
+    from_port  = 22
+    to_port    = 22
+    action     = "allow"
+    cidr_block = "169.254.99.99/15" # 169.254.0.0 - 169.255.255.255
+  }
+}
+
+# ruleid: aws-network-acl-allows-public-ingress
+resource "aws_network_acl_rule" "fail_end_outside_link_local_3" {
+  egress      = false
+  protocol    = "tcp"
+  from_port   = 22
+  to_port     = 22
+  rule_action = "allow"
+  cidr_block  = "169.254.255.255/15" # 169.254.0.0 - 169.255.255.255
+}
+
+resource "aws_network_acl" "fail_end_outside_link_local_3" {
+  vpc_id = aws_vpc.example.id
+
+  # ruleid: aws-network-acl-allows-public-ingress
+  ingress {
+    rule_no    = 1
+    protocol   = "tcp"
+    from_port  = 22
+    to_port    = 22
+    action     = "allow"
+    cidr_block = "169.254.255.255/15" # 169.254.0.0 - 169.255.255.255
+  }
+}
+
+# ruleid: aws-network-acl-allows-public-ingress
+resource "aws_network_acl_rule" "fail_end_outside_link_local_5" {
```

I made a mistake with the test case numbering:
```suggestion
resource "aws_network_acl_rule" "fail_end_outside_link_local_4" {
```